apps/www/components/preview/Modal/modalAnimation.tsx

```
@@ -0,0 +1,74 @@
+      >
+        <SelectTrigger className="w-[180px]">
+          <SelectValue placeholder="Select a animation variants" />
+        </SelectTrigger>
+        <SelectContent>
+          <SelectGroup>
+            <SelectLabel>variants</SelectLabel>
+            <SelectSeparator />
+            {(
+              Object.keys(modalVariants) as Array<keyof typeof modalVariants>
+            ).map((variantKey, index) => (
+              <SelectItem value={variantKey}>{variantKey}</SelectItem>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)